pkg/config/config.go

```diff
@@ -851,3 +852,19 @@ func stringsEq(a, b []string) bool {
 
 	return true
 }
+
+var (
+	configOnce sync.Once
+	config     *Config
+)
+
+// Default returns the default container config.  Will exit the program is something goes drastically wrong,
```

Comment:
s/is something/i**f** something/

Reply:
Removed and fixed, since this is no longer the case.